mitmproxy/test/taddons.py

```diff
@@ -105,6 +105,8 @@ def configure(self, addon, **kwargs):
             Options object with the given keyword arguments, then calls the
             configure method on the addon with the updated value.
         """
+        loader = addonmanager.Loader(self.master)
+        self.master.addons.invoke_addon(addon, "load", loader)
```

Comment:
👍👍👍, but that repeatedly calls "load", which maybe isn't optimal. How about
```python
if addon not in self.master.addons:
    self.master.addons.register(addon)
```
(this requires us to implement AddonManager.__contains__, which should be easy though)